src/drug_discovery/chemistry.py

```diff
@@ -100,6 +101,29 @@ def show_molecules_in_sdf_file(sdf_file: str | Path):
     JupyterViewer.visualize(html_content)
 
 
+@beartype
+def show_molecules_in_sdf_files(sdf_files: list[str]):
+    """show molecules in an SDF file in a Jupyter notebook using molstar"""
```

Comment:
helper merthod that combines molecules into a single SDF file, and then renders that SDF file